packages/terra-popup/src/_PopupContent.jsx

```diff
@@ -94,10 +94,19 @@ const defaultProps = {
   isWidthAutomatic: false,
 };
 
+const contextTypes = {
```

Comment:
We now recommend using `<FormattedMessage/>` and `injectIntl` over context. This PR should help: https://github.com/cerner/terra-ui/pull/132

Reply:
Added here (probably): [103fc45](https://github.com/cerner/terra-framework/pull/410/commits/103fc4576d7b40a2090f51de7e32e198446e7530)